omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -45,6 +45,17 @@
       alias /opt/opscode/version-manifest.txt;
     }
 
+    <% if node['private_chef']['data_collector']['proxy'] -%>
+    location "/data-collector/v0/" {
+      proxy_set_header        Host           $http_host;
+      proxy_set_header        X-Real-IP      $remote_addr;
+      proxy_set_header        X-Forwared-For $proxy_add_x_forwarded_for;
+      proxy_pass_request_headers on;
+      proxy_redirect off;
+      proxy_pass <%= node['private_chef']['data_collector']['root_url'] %>/data-collector/v0/;
```

Comment:
@stevendanna @stephenbm The `root_url` needs to be the fully qualified URL to data_collector endpoint that you want to POST actions to. According to the RFC, the Data Collector implementations in Chef Server and Chef Client should be agnostic from the Visibility endpoint in Chef Automate, therefore it's of my opinion that we should avoid any implementation that assumes Chef Automate and/or it's endpoint.

I think it would make sense to make the nginx location `data-collector` and leave off the API version as that relates only to the Data Collector endpoint in Chef Automate, though I'm not really all that worried if we keep it as is.  I do think that we need to change the proxy pass of the location to be only `node['private_chef']['data_collector']['root_url']` and leave it up to the user to know about appending `/data-collector/v0` in the event that you wish to forward/proxy to Automate.  Currently, If you configured the `root_url` to exclude `/data-collector/v0/` then the Chef Server would fail to forward all the Chef Server actions as the `root_url` wouldn't point to the proper path. It sounds like we should also do another pass on the documentation to make sure it's clear how a user would configure the client, server, and automate for data collection.